components/policies/development/konflux-rbac/konflux-support-nslister-access/.chainsaw-test/chainsaw-test.yaml

```diff
@@ -0,0 +1,265 @@
+---
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: verify-clusterpolicy-is-ready
+spec:
+  concurrent: false
+  description: |
+    Tests that the ClusterPolicy for generating konflux-support-nslister
+    ClusterRoleBinding is applied successfully and becomes Ready.
+  steps:
+    - name: Apply Kyverno ClusterPolicy and assert it exists
+      try:
+        - apply:
+            file: ../generate-support-nslister-clusterrolebinding-clusterpolicy.yaml
+        - assert:
+            file: chainsaw-assert-clusterpolicy.yaml
+---
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: clusterrolebinding-created-with-users
+spec:
+  concurrent: false
+  description: |
+    Tests that when a konflux-support Group is created with users,
+    a ClusterRoleBinding is generated with individual User subjects,
+    the correct label, and the view ClusterRole.
+  steps:
+    - name: given-kyverno-has-permission-on-resources
+      try:
+        - apply:
+            file: ../kyverno_rbac.yaml
+    - name: Apply Kyverno ClusterPolicy
+      try:
+        - apply:
+            file: ../generate-support-nslister-clusterrolebinding-clusterpolicy.yaml
+        - assert:
+            file: chainsaw-assert-clusterpolicy.yaml
+    - name: given-group-crd-exists
+      try:
+        - apply:
+            file: resources/group-crd.yaml
+        - sleep:
```

Comment:
We should not use sleeps. If needed we can introduce an assert step or increase the timeout for existing ones.

Reply:
changed to assertion